packages/library-base/src/commonMain/kotlin/io/realm/kotlin/dynamic/DynamicMutableRealm.kt

```diff
@@ -102,4 +102,16 @@ public interface DynamicMutableRealm : DynamicRealm {
      * @throws IllegalArgumentException if the object is invalid, frozen or not managed by Realm.
      */
     public fun delete(deleteable: Deleteable)
+
+    /**
+     * Deletes all objects of the specified class from the Realm.
+     *
+     * @param schemaClassName the class whose objects should be removed.
+     */
+    public fun delete(schemaClassName: String)
```

Comment:
What happens if you provide a className that doesn't exist? I assume we throw an `IllegalArgumentException`. In that case we should add a `@throws` to the Dokka.